packages/map-template/src/components/Wayfinding/Wayfinding.jsx

```diff
@@ -12,12 +12,17 @@ import ListItemLocation from '../WebComponentWrappers/ListItemLocation/ListItemL
 import SearchField from '../WebComponentWrappers/Search/Search';
 import { snapPoints } from '../../constants/snapPoints';
 import { usePreventSwipe } from '../../hooks/usePreventSwipe';
+import handleGooglePlaces from "../Map/GoogleMapsMap/GooglePlacesHandler";
+import GooglePlaces from '../../assets/google-places.png';
+import { mapTypes } from "../../constants/mapTypes";
 
 const searchFieldIdentifiers = {
     TO: 'TO',
     FROM: 'FROM'
 };
 
+const markerIcon = "data:image/svg+xml,%3Csvg width='10' height='10' viewBox='0 0 14 20' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M7 0C3.13 0 0 3.13 0 7C0 12.25 7 20 7 20C7 20 14 12.25 14 7C14 3.13 10.87 0 7 0ZM7 9.5C5.62 9.5 4.5 8.38 4.5 7C4.5 5.62 5.62 4.5 7 4.5C8.38 4.5 9.5 5.62 9.5 7C9.5 8.38 8.38 9.5 7 9.5Z' fill='black' fill-opacity='0.88'/%3E%3C/svg%3E%0A"
```

Comment:
Maybe the name should indicate that it is used for Google Places.

Reply:
Renamed variable. ✅